src/test/scala/uk/gov/nationalarchives/tdr/api/routes/TransfersAgreementRouteSpec.scala

```diff
@@ -23,18 +24,19 @@ class TransfersAgreementRouteSpec extends AnyFlatSpec with Matchers with TestReq
   case class GraphqlMutationData(data: Option[AddTransferAgreement], errors: List[GraphqlError] = Nil)
   case class GraphqlQueryData(data: Option[TransferAgreement], errors: List[GraphqlError] = Nil)
   case class TransferAgreement(
-                                consignmentid: Option[UUID] = None,
+                                consignmentId: Option[UUID] = None,
                                 allPublicRecords: Option[Boolean] = None,
                                 allCrownCopyright: Option[Boolean] = None,
                                 allEnglish: Option[Boolean] = None,
                                 appraisalSelectionSignedOff: Option[Boolean] = None,
-                                sensitivityReviewSignedOff: Option[Boolean] = None,
-                                transferAgreementId: Option[UUID] = None
+                                initialOpenRecords: Option[Boolean] = None,
```

Comment:
I agree with the casing changes, but are they backwards-compatible? Does this imply that the API client will need to be updated too?

Reply:
The casing changes now match the what is in the TransferAgreementFields.TransferAgreement case class.

Reply:
Then how did these tests pass before? I thought this class was used for deserializing the response in the test. 🤔

Reply:
Not sure, but if I revert the back to `consignmentid` the test fail. Curious